packages/ui/src/ui-component/cards/ItemCard.jsx

```diff
@@ -106,47 +140,92 @@ const ItemCard = ({ data, images, onClick }) => {
-                                </Typography>
-                            )}
-                        </Box>
-                    )}
+                    <Box
+                        sx={{
+                            display: 'flex',
+                            alignItems: 'center',
+                            justifyContent: 'start',
+                            gap: 1,
+                            mt: 2 // Add margin-top to separate from description
+                        }}
+                    >
+                        {images && (
+                            <Box
+                                sx={{
+                                    display: 'flex',
+                                    alignItems: 'center',
+                                    justifyContent: 'start',
+                                    gap: 1
+                                }}
+                            >
+                                {images.slice(0, images.length > 3 ? 3 : images.length).map((img, index) => (
+                                    <Tooltip key={img} title={nodeTypes && nodeTypes[index]} arrow>
+                                        <Box
+                                            sx={{
+                                                width: 30,
+                                                height: 30,
+                                                borderRadius: '50%',
+                                                backgroundColor: customization.isDarkMode
+                                                    ? theme.palette.common.white
+                                                    : theme.palette.grey[300] + 75
+                                            }}
+                                        >
+                                            <img
+                                                style={{ width: '100%', height: '100%', padding: 5, objectFit: 'contain' }}
+                                                alt=''
+                                                src={img}
+                                            />
+                                        </Box>
+                                    </Tooltip>
+                                ))}
+                                {images.length > 3 && (
+                                    <Typography sx={{ alignItems: 'center', display: 'flex', fontSize: '.9rem', fontWeight: 200 }}>
+                                        + {images.length - 3} More
+                                    </Typography>
+                                )}
+                            </Box>
+                        )}
+                        {data.category && (
+                            <Box sx={{ display: 'flex', flexWrap: 'wrap', gap: 1 }}>
+                                {data.category.split(';').map((tag, index) => (
+                                    <Chip
+                                        key={index}
```

Comment:
key shoudl be more unique.   I usually do something like ``chip-category-${tag}${index}``